src/main/java/Ian.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Scanner;
+
+public class Ian {
+    private static void printSeparator() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void mark_or_unmark(Boolean toMark, String input, Task[] tasks) {
+        char lastChar = input.charAt(input.length() - 1);
+        int  index = (Integer.parseInt(String.valueOf(lastChar))) - 1;
```

Comment:
Ideally keep whitespaces between words to just 1 character, so:

```suggestion
        int index = (Integer.parseInt(String.valueOf(lastChar))) - 1;
```

Just a minor suggestion :)